tests/unit/s2n_pq_kem_test.c

```diff
@@ -50,6 +54,19 @@ int main()
 {
     BEGIN_TEST();
 
+#if defined(OPENSSL_IS_AWSLC)
+    const unsigned long lc_vers = awslc_api_version_num();
+# else
+    const unsigned long lc_vers = SSLeay();
+#endif
+
+    /* If using non-FIPS AWS-LC >= v1.4.0 (API vers. 20), expect Kyber512 KEM from AWS-LC*/
```

Comment:
You're very confident that this version is correct? Since this is just a sanity check and the feature probe actually turns on the feature, definitely err on the side of caution here: we never want this to fail, but it's fine / expected that it doesn't catch all environments where this is true.

Reply:
yeah, i'm pretty sure this statement is correct. here's the [1.4.0 release](https://github.com/aws/aws-lc/releases/tag/v1.4.0) at [commit `cfbc38b`](https://github.com/aws/aws-lc/commit/cfbc38bfbd72cfb1a680d0483381b2e93b9a3e73). you can see the KEM stuff [here](https://github.com/aws/aws-lc/blob/cfbc38bfbd72cfb1a680d0483381b2e93b9a3e73/include/openssl/evp.h#L872-L902) and the API version [here](https://github.com/aws/aws-lc/blob/cfbc38bfbd72cfb1a680d0483381b2e93b9a3e73/include/openssl/base.h#L207).